specs/monitor_build.spec

```
@@ -115,10 +127,10 @@ exe = EXE(
     runtime_tmpdir=None,
     console=True,
     disable_windowed_traceback=False,
-    argv_emulation=False,
-    target_arch=None,
+    argv_emulation=True if sys.platform == 'darwin' else False,  # macOS 需要开启
```

RuboCop (1.81.7)

[fatal] unexpected token kIF_MOD
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)